src/Kerbero.Domain/NukiAuthentication/Models/AccountMapper/NukiAccountMapper.cs

```diff
@@ -32,7 +32,7 @@ public static NukiAccountAuthenticatedResponseDto MapToAuthenticatedResponse(Nuk
 		{
 			NukiAccountId = nukiAccount.Id,
 			ClientId = nukiAccount.ClientId,
-			Token = nukiAccount.Token
+			Token = nukiAccount.Token!
```

Comment:
```suggestion
			Token = nukiAccount.Token
```
niente null assertion sulle assegnazioni, il problema è da un altra parte

Reply:
e se fossi sicuro che in questo contesto non sia nullo?